docs/src/content/docs/getting-started.mdx

```diff
@@ -5,33 +5,33 @@ description: Learn how to start building your next documentation site with Starl
 
 import { Tabs, TabItem } from '@astrojs/starlight/components';
 
-## Creating a new project
-
 Starlight is a full-featured documentation theme built on top of the [Astro](https://astro.build) framework.
+This guide will help you get started with a new project.
```

Comment:
```suggestion
This guide will help you get started with a new project.
See the [manual installation instructions](/manual-setup/) to add the Starlight integration to an existing project.
```
What do you think about something like this?

In Astro docs, manual setup is an analogous process (you still do create a new Astro project, just manually). In this case, "manual" really does mean add to an existing project vs create a new Starlight project, just by hand. You need an existing site already.

I *might* even suggest that that page title should more reflect Add to an Existing project? But, I think it's helpful to explicitly call out a) that you can and b) that that's what you'll find under "manual installation"

Reply:
Yeah, I like this!

Our manual setup guide does technically work for “`create astro` with the empty template, then add Starlight”, and I think of it as the place to understand better the different components of a Starlight set-up. But yeah, the main user scenario it serves is the “Add to existing site” one.

Reply:
(Noting for transparency that I edited your suggestion to change “manual install” to “manual setup”, which is how we refer to the guide generally as it’s more than “installing”.)